src/index.ts

```diff
@@ -1,3 +1,3 @@
 export * as commands from './commands/index.js';
 export * as message from './utils/message.js';
-//export * as utils from './utils/index.js';
+export * as constants from './constants/index.js';
```

Comment:
utils most likely will be used in the next mr

Reply:
fixed